src/components/SearchForm/SearchFormOptions.tsx

```diff
@@ -0,0 +1,145 @@
+import lungRadiationCodes from 'src/queries/mockData/lungRadiationCodes.json';
+import lungSurgeryCodes from 'src/queries/mockData/lungSurgeryCodes.json';
+import MultipleMyelomaCancerTypeCodes from 'src/queries/mockData/MultipleMyelomaCancerTypeCodes.json';
+import MultipleMyelomaMedicationCodes from 'src/queries/mockData/MultipleMyelomaMedicationCodes.json';
+import prostateCancerTypeCodes from 'src/queries/mockData/prostateCancerTypeCodes.json';
+
+//import type { NamedSNOMEDCode } from '@/utils/fhirConversionUtils';
+
+/*for ( var i = 0 ; i < lungCancerTypeCodes.entry.length; i++){
+    console.log(lungCancerTypeCodes.entry[i].display)
+
+}*/
+export const breastCancerStages = [0, 1, 2.1, 2.2, 3.1, 3.2, 3.3, 4];
+//export const breastCancer_cancerTypeOptions=buildOptionListfromJSON(breastcancerCodes);
+export const breastCancerMedications = '';
+export const breastCancerSurgeryCodes = '';
+export const breastCancer_proceduresOptions = '';
+export const breastCancerBiomarkers = '';
+export const breastCancer_biomarkersOptions = '';
+export const breastCancerRadiationCodes = '';
+export const breastCancerSubTypes = '';
+export const breastCancerCodes = '';
+export type cancerTypeDetails = {
+  category: string;
+  cancerCodes: NamedSNOMEDCode[];
+  cancerSubtype: NamedSNOMEDCode[];
+  biomarkers: NamedSNOMEDCode[];
+  surgeryCodes: NamedSNOMEDCode[];
+  medications: NamedSNOMEDCode[];
+  stages: NamedSNOMEDCode[];
+  radiationCodes: NamedSNOMEDCode[];
+};
+export type cancerTypeObj = {
+  lung: cancerTypeDetails;
+  breast: cancerTypeDetails;
+  prostate: cancerTypeDetails;
+  brain: cancerTypeDetails;
+  colon: cancerTypeDetails;
+  mm: cancerTypeDetails;
+};
+
+const tmp: cancerTypeObj = {
+  lung: buildCancerCodeJSON(
+    'lung',
+    lungCancerTypeCodes,
+    lungCancerSubTypeCodes,
+    lungMedicationCodes,
+    lungRadiationCodes,
+    lungSurgeryCodes,
+    breastCancerStages,
+    breastCancerBiomarkers
+  ),
+  colon: buildCancerCodeJSON(
+    'colon',
+    colonCancerTypeCodes,
+    '',
+    colonMedicationCodes,
+    colonRadiationCodes,
+    colonSurgeryCodes,
+    breastCancerStages,
+    ''
+  ),
+  brain: buildCancerCodeJSON(
+    'brain',
+    brainCancerTypeCodes,
+    brainCancerSubTypeCodes,
+    brainMedicationCodes,
+    brainRadiationCodes,
+    brainSurgeryCodes,
+    breastCancerStages,
+    brainBiomarkerCodes
+  ),
+  prostate: buildCancerCodeJSON('prostate', prostateCancerTypeCodes, '', '', '', '', breastCancerStages, ''),
+  breast: buildCancerCodeJSON(
+    'breast',
+    breastCancerCodes,
+    breastCancerSubTypes,
+    breastCancerMedications,
+    breastCancerRadiationCodes,
+    breastCancerSurgeryCodes,
+    breastCancerStages,
+    breastCancerBiomarkers
+  ),
+  mm: buildCancerCodeJSON(
+    'mm',
+    MultipleMyelomaCancerTypeCodes,
+    '',
+    MultipleMyelomaMedicationCodes,
+    '',
+    '',
+    breastCancerStages,
+    ''
+  ),
+};
+
+export const cancerTypeDetails: cancerTypeObj = tmp;
```

Comment:
I'm unclear why the `const tmp` and why not just `export const cancerTypeDetails: cancerTypeObj`.

Reply:
removed tmp variable as suggested.